utilities/rocAL/rocal_external_source/README.md

```diff
@@ -0,0 +1,25 @@
+# rocal_external_source application
+This application demonstrates a basic usage of rocAL's C API to load images from external source and add augmentations and and displays the output images.
+
+## Build Instructions
+
+### Pre-requisites
+* Ubuntu Linux, [version `18.04` or later](https://www.microsoft.com/software-download/windows10)
+* rocAL library (Part of the MIVisionX toolkit)
+* [OpenCV 3.1](https://github.com/opencv/opencv/releases) or higher
+* Radeon Performance Primitives (RPP)
```

Comment:
Please rename Radeon performance primitives to ROCm performance primitives

Reply:
@swetha097 Please check this

Reply:
Done